rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml

```diff
@@ -9,15 +9,25 @@ date: 2022/01/16
 references:
     - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1218.007/T1218.007.md
+    - https://twitter.com/_st0pp3r_/status/1583914244344799235
 logsource:
     category: process_creation
     product: windows
 detection:
     selection:
         Image|endswith: '\msiexec.exe'
-        CommandLine|contains|all:
+        CommandLine|contains:
             - ' /i'
+            - ' -i'
+            - ' /package'
+            - ' -package'
+            - ' /a'
+            - ' -a'
+            - ' /j'
+            - ' -j'
+        CommandLine|contains:
             - ' /q'
+            - ' -q'
```

Comment:
Can not have 2 `CommandLine|contains` in the same selection
```yaml
    selection_msi:
        Image|endswith: '\msiexec.exe'
        CommandLine|contains:
            - ' /i'
            - ' -i'
            - ' /package'
            - ' -package'
            - ' /a'
            - ' -a'
            - ' /j'
            - ' -j'
     selection_quiet:
        CommandLine|contains:
            - ' /q'
            - ' -q'
    condition: all of selection_*
```
and
```yaml
date: 2022/01/16
modified: 2022/10/23
```

Reply:
changes made. sorry about that I am usually more careful.